DH_Construction/Classes/DHConstruction_PlatoonHQ.uc

```diff
@@ -339,6 +339,7 @@ defaultproperties
     ProgressMax=9
     SupplyCost=750
     MinDamagetoHurt=50
+    OptimalTraceOffset=(Z=40.0)
```

Comment:
My only recommendation is that this gets renamed to `ExplosionDamageTraceOffset`, since `Optimal` raises the question of "optimal for what"?

Reply:
I spent an embarrassing amount of time debating with myself on how to name it. Opted for `Optimal` because it can be used for any kind of tracing and get consistent results. But yea, `ExplosionDamageTraceOffset` is more specific and that's all it's used for now anyways.

I'll rename it.